src/background.js

```diff
@@ -3,21 +3,24 @@ const FACEBOOK_CONTAINER_NAME = "Facebook";
 const FACEBOOK_CONTAINER_COLOR = "blue";
 const FACEBOOK_CONTAINER_ICON = "briefcase";
 const FACEBOOK_DOMAINS = [
-  "facebook.com", "www.facebook.com", "facebook.net", "fb.com", 
+  "facebook.com", "www.facebook.com", "facebook.net", "fb.com",
```

Comment:
The spaces were left intentionally, you shouldn't remove them.

Reply:
Also merge the two commits into one commit for better history management.

Reply:
spaces removal was by IDE, will fix. merging the commits.

Reply:
@nemoload - can you tell me why do we have spaces in the end only in some lines and not all of the lines ( example - line 7,8,11,15,17  don't have spaces in end?